src/coreclr/jit/emitarm64.h

```diff
@@ -28,7 +28,7 @@ void emitDispInsHelp(
 void emitDispLargeJmp(
     instrDesc* id, bool isNew, bool doffs, bool asmfm, unsigned offset, BYTE* pCode, size_t sz, insGroup* ig);
 void emitDispInst(instruction ins);
-void emitDispImm(ssize_t imm, bool addComma, bool alwaysHex = false);
+void emitDispImm(ssize_t imm, bool addComma, bool alwaysHex = false, bool isAddrOffset = false);
```

Comment:
Would be nice to fix this for ARM as well.

Reply:
Sure, will do. Thanks for reminding me.